pkg/amazon/backend/testdata/simple/simple-cloudformation-conversion.golden

```diff
@@ -275,16 +271,6 @@
         "NetworkMode": "awsvpc",
         "RequiresCompatibilities": [
           "FARGATE"
-        ],
-        "Tags": [
```

Comment:
Was this intentional?

Reply:
I ran tests with `-test.update-golden`. I assume master is borken then :P

Reply:
indeed : https://github.com/docker/ecs-plugin/runs/952407798